src/photo.rs

```diff
@@ -88,6 +88,19 @@ impl Photo
 	}
 }
 
+impl Photo
+{
+	pub fn id(&self) -> u32
```

Comment:
This also, see above, might be just as well to call this `get_id`

Reply:
Cf. Previous answer